<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1336</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1336</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/picture/maphard_init.png);</string>
  </property>
  <widget class="QPushButton" name="musicon">
   <property name="geometry">
    <rect>
     <x>1268</x>
     <y>18</y>
     <width>45</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/music_on.png);
border:1px solid rgba(255,255,255,0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="musicoff">
   <property name="geometry">
    <rect>
     <x>1268</x>
     <y>17</y>
     <width>24</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/music_off.png);border:1px solid rgba(255,255,255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>930</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#setting{background-image: url(:/new/prefix1/picture/setting_dark.png);border:1px solid rgba(255,255,255,0);}QPushButton#setting:hover{background-image: url(:/new/prefix1/picture/setting_light.png);border:1px solid rgba(255,255,255,0);}QPushButton#setting:pressed{background-image: url(:/new/prefix1/picture/setting_light.png);border:1px solid rgba(255,255,255,0);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>0</y>
     <width>210</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/timer.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>0</y>
     <width>35</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>0</y>
     <width>35</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>0</y>
     <width>35</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>0</y>
     <width>35</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>1012</x>
     <y>241</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/fire.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelImg1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>620</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/ice.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue1">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>880</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>880</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/button.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="block">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/block.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blue6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>635</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/blue diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red5">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>105</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="red6">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>635</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/red diamond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>button</zorder>
  <zorder>block</zorder>
  <zorder>musicon</zorder>
  <zorder>musicoff</zorder>
  <zorder>setting</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>labelImg</zorder>
  <zorder>labelImg1</zorder>
  <zorder>blue1</zorder>
  <zorder>blue2</zorder>
  <zorder>blue3</zorder>
  <zorder>red1</zorder>
  <zorder>red2</zorder>
  <zorder>red3</zorder>
  <zorder>blue4</zorder>
  <zorder>blue5</zorder>
  <zorder>blue6</zorder>
  <zorder>red4</zorder>
  <zorder>red5</zorder>
  <zorder>red6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
